### PR TITLE
acts: init at 1.3

### PR DESCRIPTION
**Pull request by asymmetric**
###### Motivation for this change

An alternative to [tarsnapper](https://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/backup/tarsnapper/default.nix) and the [NixOS module](ithttps://github.com/NixOS/nixpkgs/blob/master/pkgs/tools/backup/tarsnapper/default.nix).

It uses [logger](https://linux.die.net/man/1/logger), does it mean I need to be declared a runtime dependency to `utilLinuxMinimal`, or is it somehow already included on all Linux builds?

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

**Comment by asymmetric** [comment]
Also, not sure how to deal with configuration, since the paths are hardcoded to `/etc/acts.conf` and `/usr/local/etc/acts.conf`.

**Comment by Lassulus** [comment]
I would test this after the merge conflict is fixed.

**Comment by asymmetric** [comment]
@Lassulus conflicts fixed.

I'm still unsure if this warrants a NixOS module, with config file generation, or if it the package is enough for now, and we can expect users to add a file to `/etc/acts.conf` manually.

File: pkgs/tools/backup/acts/default.nix
Comment (Lassulus) [inline pkgs/tools/backup/acts/default.nix]:
```suggestion
  pname = "acts";
```

**Comment by Lassulus** [comment]
It seems to get the config also from the current working directory (CWD) so I guess a package is sufficient for now. But if you want to also add a service, I won't stop you :)

File: pkgs/tools/backup/acts/default.nix
Comment (Lassulus) [inline pkgs/tools/backup/acts/default.nix]:
```suggestion
    homepage = "https://github.com/alexjurkiewicz/acts";
```

**Comment by Lassulus** [comment]
whupps, somehow missed the updates, still good for merge?

**Comment by asymmetric** [comment]
@Lassulus please hold off on merging, this could be made to use `DynamicUser`, `LogsDirectory` etc.

**Review by ryantm — CHANGES_REQUESTED** [review]
asymmetric wants to update this to use DynamicUser

**Comment by asymmetric** [comment]
Not using this at the moment.

**Pull request closed without merging**
